modules/030-cloud-provider-yandex/images/cloud-metrics-exporter/werf.inc.yaml

```diff
@@ -0,0 +1,50 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+from: {{ env "BASE_ALPINE" }}
+import:
+- artifact: {{ .ModuleName }}/{{ .ImageName }}-artifact
+  add: /src/exporter
+  to: /exporter
+  after: setup
+docker:
+  ENTRYPOINT: [ "/exporter"]
+---
+artifact: {{ .ModuleName }}/{{ .ImageName }}-artifact
+from: {{ env "BASE_GOLANG_16_ALPINE" }}
+git:
+- add: /modules/030-{{ .ModuleName }}/images/{{ .ImageName }}
+  to: /src
+  excludePaths:
+    - "**/*.md"
+    - "**/*.yaml"
+    - hack
+  stageDependencies:
+    install:
+      - go.mod
+      - go.sum
+    setup:
+      - go.mod
+      - go.sum
+      - "**/*.go"
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+
+ansible:
```

Comment:
If I remember correctly, we are trying to avoid using ansible. We are using shell. But it is not a problem, it is for information only

Reply:
Ok, i got it, but can i not fix it?

Reply:
yes, you should not do anything, it's only for information